### PR TITLE
read accounts into env during deployment, to allow referincing to acc…

### PR DESCRIPTION
**Pull request by RnkSngh**
this PR reads all accounts from the accounts.spec file into the env. This allows us to reference multisig safe addresses from accounts in the account spec, rather than having to hard code everything. Specifically, this is useful for knowing which account to transfer ownership to during ownership transfers

### CODE REVIEW

File: src/utils/io.ts [hunk 1]
```diff
@@ -239,6 +239,20 @@ export async function writeDeployedContractToFile(
   });
 }
 
+// Read existing accounts into env
+export async function readAccountsIntoEnv(
+  env: any,
+  accountRegistry: AccountRegistry
+) {
+  console.log(env, accountRegistry.keys());
```
Comment (dshiell) [inline src/utils/io.ts:247]: pub keys right?

**Review by dshiell — APPROVED** [review]
lgtm!

File: src/utils/io.ts (hunk 1 above)
Comment (RnkSngh) [reply src/utils/io.ts:247]: err whoops, yes it only logs pub keys, but I should get rid of this - good catch!

Comment (RnkSngh) [reply src/utils/io.ts]: removed now!

**Pull request merged**